src/test/scenarios/managed-network/input/managedNetwork.json

```diff
@@ -89,7 +89,8 @@
             "schema": {
               "$ref": "#/definitions/ManagedNetwork"
             },
-            "description": "Parameters supplied to the create/update a Managed Network Resource"
+            "description": "Parameters supplied to the create/update a Managed Network Resource",
+            "x-ms-client-flatten": true
```

Comment:
can we add "x-ms-client-flatten" to managed network in Azure REST API specification?
I am worried it may cause breaking change in the SDK, so I am not sure about it

Reply:
+1, is it possible to put this in autorest's directive?

---
In reply to: [382984586](https://github.com/Azure/autorest.az/pull/131#discussion_r382984586) [](ancestors = 382984586)

Reply:
just realize this is in our test. to confirm, this is jus ta copy and won't impact real spec, right?

Reply:
yes, this is our test.

Reply:
but we should backpropagate that to swagger somehow, we shouldn't be using our own versions of swagger to generate real command modules